.env.example

```diff
@@ -4,4 +4,6 @@ GATSBY_NETWORK="rinkeby"
 #GATSBY_INFURA_PROJECT_ID="xxx"
 #GATSBY_MARKET_FEE_ADDRESS="0xxx"
 #GATSBY_ANALYTICS_ID="xxx"
-#GATSBY_PORTIS_ID="xxx"
\ No newline at end of file
+#GATSBY_PORTIS_ID="xxx"
+#GATSBY_ALLOW_FRE_PRICING="true"
```

Comment:
consistency, if we call it `GATSBY_ALLOW_DYNAMIC_PRICING` then the other one should be `GATSBY_ALLOW_FIXED_PRICING`. Only insiders know what `fre` stands for